Rakefile

```diff
@@ -35,6 +35,10 @@ namespace :docker do
   end
 end
 
+task :proto do
+  sh "bundle exec grpc_tools_ruby_protoc -I protos/ --ruby_out=./protos --grpc_out=./protos protos/*.proto"
```

Comment:
Does this create ruby libs for the proto files?

Reply:
It does!

Reply:
Let's name this `generate_protos` or something like that